pkg/services/publicdashboards/api/api.go

```diff
@@ -171,7 +218,8 @@ func (api *Api) handleError(ctx context.Context, code int, message string, err e
 		return response.Error(publicDashboardErr.StatusCode, publicDashboardErr.Error(), publicDashboardErr)
 	}
 
-	// handle dashboard errors as well
+	// NOTE this code may never hit as it always casts to public dashboard error
```

Comment:
we're returning dashboard errors.
For example, check Update method in service
```suggestion
	
```

Reply:
Thanks. This comment was breadcrumb. Fixed a bug that was affected all API endpoints : )